data_model/master/clusters/GeneralCommissioningCluster.xml

```diff
@@ -65,7 +65,7 @@ Davis, CA 95616, USA
   </clusterIds>
   <classification hierarchy="base" role="utility" picsCode="CGEN" scope="Node"/>
   <features>
-    <feature bit="0" code="TC" name="Terms" summary="Supports Terms &amp; Conditions acknowledgement">
+    <feature bit="0" code="TC" name="Terms and Conditions" summary="Supports Terms and Conditions acknowledgement">
```

Comment:
data_model files are scraped from the spec and should not be manually updated.  please remove these changes from the PR.

I should also check why the CI checker isn't flagging these...

Reply:
I reverted the changes to this file.